src/routes/web.php

```diff
@@ -56,7 +56,14 @@
         Route::put('/{trip}', 'update')
             ->name('update');
 
+        // Add report for the trip
+        Route::put('/{trip}/add-report', 'addReport')
```

Comment:
nebavili sme sa o route na pridanie poznamky? ci si len zle pamatam. lebo podla mna to malo byt add note, add report ide cez rovnaky formular ako obycajny edit a to sme zatial neriesili, ze by sa delilo

Reply:
Ano ano, to som si nevsimol, uz som to mergol, idem to rychlo opravit

Reply:
Malo by byt fixnute v #81